crates/uv-settings/src/lib.rs

```diff
@@ -59,6 +59,20 @@ impl FilesystemOptions {
         }
     }
 
+    pub fn system() -> Result<Option<Self>, Error> {
+        if let Some(file) = system_config_file() {
```

Comment:
You can use the same pattern here as from line 37, to save some indent by putting the None up top.